packages/ai-core/src/common/agent-service.ts

```diff
@@ -131,14 +131,14 @@ export class AgentServiceImpl implements AgentService {
         return this._agents;
     }
 
-    enableAgent(agentId: string): void {
+    async enableAgent(agentId: string): Promise<void> {
```

Comment:
Could you also update the interface definitions higher up in the file so they reflect the new async signatures? TIA!